### ISSUE TITLE
Add short-keys for the buttons

### ISSUE THREAD

**Issue by Nudin** [Nudin/makesense#4]
>  Could you add some common shortcuts for each button please, e.g. "s", "r", and "n" to quickly perform the command without the mouse? And also add that as a tooltip for each of the buttons for easy discoverability?

**Comment by DanD16** [comment]
The other pull request about this seems inactive, so I went ahead and tried to solve this issue. I have a complete solution for this. Since the other guy supposed to work on it I didn't want to be disrespectful and just pull request my solution, however if you give me permission, I'm more than happy to upload it today or tomorrow!

**Comment by Nudin** [comment]
No problem from my side. Pinging @dzarmola to let him know about this.

**Comment by DanD16** [comment]
@Nudin So I've encountered an interesting problem here. My Javascript part is perfectly working, no problem, however the tooltip completely breaks the buttons. I tried the "title" in HTML as well as to create a tooltip in pure CSS, and for whatever reason as soon as the tooltip added in any way, the button instantly breaks and doesn't work anymore (doesn't respond to simple clicks either).

Since I have absolutely no idea why, I might add a new symbol, like a question mark with a tooltip about shortkeys.
